### PR TITLE
Contact case details choose case

### PR DESCRIPTION
**Pull request by adriandumavitagroup**

### CODE REVIEW

File: src/app/shared/modals/choose-case-modal/choose-case-modal.component.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<div mat-dialog-title>
+  <h2>{{ 'Select source case' | translate }}</h2>
```
Comment (sandorbanvitagroup) [inline src/app/shared/modals/choose-case-modal/choose-case-modal.component.html]: Please add translation for this

File: src/app/shared/modals/choose-case-modal/choose-case-modal.component.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  <div mat-dialog-actions>
+    <button mat-icon-button mat-dialog-close class="button-dialog-close">
+      <mat-icon class="material-icons-round">highlight_off</mat-icon>
+    </button>
+    <button mat-stroked-button color="primary" class="cancel-btn" mat-dialog-close>
+      {{ 'Cancel' | translate }}
```
Comment (sandorbanvitagroup) [inline src/app/shared/modals/choose-case-modal/choose-case-modal.component.html]: Please add translation for this

File: src/app/shared/modals/choose-case-modal/choose-case-modal.component.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    </button>
+    <button mat-flat-button color="primary" class="confirm-btn" (click)="confirm()">
+      {{ 'Confirm' | translate }}
```
Comment (sandorbanvitagroup) [inline src/app/shared/modals/choose-case-modal/choose-case-modal.component.html]: Please add translation for this

**Review by danielrus — DISMISSED** [review]

File: src/app/shared/modals/choose-case-modal/choose-case-modal.component.html (hunk 1 above)
Comment (adriandumavitagroup) [reply src/app/shared/modals/choose-case-modal/choose-case-modal.component.html]: done

File: src/app/shared/modals/choose-case-modal/choose-case-modal.component.html (hunk 3 above)
Comment (adriandumavitagroup) [reply src/app/shared/modals/choose-case-modal/choose-case-modal.component.html]: done

File: src/app/shared/modals/choose-case-modal/choose-case-modal.component.html (hunk 2 above)
Comment (adriandumavitagroup) [reply src/app/shared/modals/choose-case-modal/choose-case-modal.component.html]: done

**Review by danielrus — APPROVED** [review]

**Review by sandorbanvitagroup — APPROVED** [review]

**Pull request merged**